membership-attribute-service/app/models/Attributes.scala

```diff
@@ -18,17 +18,19 @@ object ContentAccess {
 
 case class Attributes(
                        UserId: String,
-                       Tier: String,
+                       Tier: Option[String] = None,
                        MembershipNumber: Option[String],
                        AdFree: Option[Boolean] = None,
                        CardExpirationMonth: Option[Int] = None,
-                       CardExpirationYear: Option[Int] = None) {
-  require(Tier.nonEmpty)
+                       CardExpirationYear: Option[Int] = None,
+                       Contributor: Option[String] = None) {
```

Comment:
`contributor` should be a boolean? or is this actually `contributorFrequency`? or `contributorAmount`?